tetris-PRG.info

```diff
@@ -816,7 +816,15 @@ LABEL { ADDR $9B96; NAME "@checkForBorrow"; };
 LABEL { ADDR $9BA6; NAME "@gameTypeA"; };
 LABEL { ADDR $9BA8; NAME "incrementLines"; };
 LABEL { ADDR $9BFE; NAME "addHoldDownPoints"; };
+LABEL { ADDR $9C18; NAME "@noLowOverflow"; };
```

Comment:
Yeah, I think I might have named these (not thinking about it too long) `@score0High`, `@markRenderOutOfDate`, and instead of `@noScore0HighOverflow` maybe `@score1Low`. `@pointsLoop` is interesting and it does seem a bit hard to call out that it is multiplying by level, so maybe a comment is easiest there. Or maybe `@loopForCurrentLevel`. "lines" or "level" is probably better than "points" in the name, so `@linesLoop` or `@levelLoop` would be fair game. Since the entire function is about points, just `@loop` would be fine if another name wasn't forthcoming.

Your names are fine too, but they cater to different usages. Naming the code at the label provides an overview of the function at-a-glance. Describing based on the jump makes it easier to understand those individual lines of code. For long stuff like this, I've found details like the carry are pretty easy to work out compared to the repeated effort of finding the particular part of code you are interested in.

But that sort of stuff isn't reason enough to change it. I'd only end up changing the names at this point if I experience repeated issue with the current names.

Reply:
All of this is FYI, and just in case you are interested. Not really a critique of your names.